### PR TITLE
RELATED: RAIL-2924 - Add 'X-Requested-With' header to dev proxy

### PR DESCRIPTION
**Pull request by lupko**

### CODE REVIEW

**Comment by yenkins** [comment]
## The code coverage is **the same** after your change 🤟
This is a good thing if you are not sure ;)
#### Report (compared to build [#178](https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/178/))
Covered lines: **26.8%**, change: **0%**
Covered branches: **25.93%**, change: **0%**
#### Coverage report
https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/179/artifact/ci/results/coverage/lcov-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/179/ : SUCCESS in 36s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-oss-check/148/ : SUCCESS in 7s

**Comment by yenkins** [comment]
## The code coverage is **the same** after your change 🤟
This is a good thing if you are not sure ;)
#### Report (compared to build [#178](https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/178/))
Covered lines: **26.8%**, change: **0%**
Covered branches: **25.93%**, change: **0%**
#### Coverage report
https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/180/artifact/ci/results/coverage/lcov-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/180/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-oss-check/149/ : SUCCESS in 6s

File: bootstrap/src/setupProxy.js [hunk 1]
```diff
@@ -13,6 +13,8 @@ module.exports = function(app) {
             headers: {
                 host: domain.replace(/https:\/\//, ''),
                 origin: null,
+                // This is essential for Tiger backends. To ensure 401 flies when not authenticated and using proxy
+                "X-Requested-With": "XMLHttpRequest",
```
Comment (no23reason) [inline bootstrap/src/setupProxy.js:17]: Is this safe for bear? Can we avoid mentionning tiger in a file that will be used in bear too?

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
## The code coverage is **the same** after your change 🤟
This is a good thing if you are not sure ;)
#### Report (compared to build [#178](https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/178/))
Covered lines: **26.8%**, change: **0%**
Covered branches: **25.93%**, change: **0%**
#### Coverage report
https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/181/artifact/ci/results/coverage/lcov-report/index.html

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-merge-compliance-check/65/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/181/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-fossa-scanning-zuul-el7/30/ : SUCCESS in 17s (non-voting)

**Pull request merged**